Memory_Flash_NAND.lib

```diff
@@ -0,0 +1,434 @@
+EESchema-LIBRARY Version 2.4
+#encoding utf-8
+#
+# ONFI_NV-DDR2x8_BGA-132
+#
+DEF ONFI_NV-DDR2x8_BGA-132 U 0 20 Y Y 2 L N
+F0 "U" 0 0 50 H V C CNN
+F1 "ONFI_NV-DDR2x8_BGA-132" 0 -100 50 H V C CNN
+F2 "Package_BGA:BGA-132_12x18mm_Layout11x17_P0.5mm" 0 -200 50 H I C CNN
```

Comment:
The package name appears to be wrong. The pitch is 0.8 in the footprint is correct but the filename suggest is it P0.5mm

Reply:
This comment was intended for the BGA63 package